.gitignore

```diff
@@ -0,0 +1 @@
+/elm-stuff
```

Comment:
Let's not have gitignore versioned. This lets everyone have whatever they need in it.

E.g., in your own gitignore you can list `.gitignore` itself, so that Git doesn't see it.

Reply:
I can do that!